lib/modelManagers/usermodel.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unused-vars */
+import user from '../../database/models/user';
```

Comment:
pls refactor the import statement to require because it will give issues to those depending on this file

Reply:
alright, done.